### ISSUE TITLE
--loopdelay-ms with --loop=0 not working

### ISSUE THREAD

**Issue by 2Belette** [appneta/tcpreplay#542]
I would like to wait between loop but looping indefinitely on the same time.
--loopdelay-ms is working when --loop is different than 0.

It is normal behavior?

**Comment by fklassen** [comment]
Yes, that is the normal behavior.

**Comment by 2Belette** [comment]
Would it make sense to allow the option for --loop=0 ? 
(at least on my use case it would)

**Comment by fklassen** [comment]
It's not clear from you discussion what the behavior is, and what you are actually doing. If you are saying it does not wait --loopdelay-ms when --loop=0, then that is a bug that needs to be reported.

If you feel this is a bug, please report version numbers, and exact examples of commands that work and that do not work.

**Comment by 2Belette** [comment]
thanks @fklassen yes it does not wait --loopdelay-ms when --loop=0, so we both agree it is a bug this is the reason why I opened this ticket :)
Version : 
```
tcpreplay --version
tcpreplay version: 4.2.6 (build git:v4.2.6)
Copyright 2013-2017 by Fred Klassen <tcpreplay at appneta dot com> - AppNeta
Copyright 2000-2012 by Aaron Turner <aturner at synfin dot net>
The entire Tcpreplay Suite is licensed under the GPLv3
Cache file supported: 04
Compiled against libdnet: 1.12
Compiled against libpcap: 1.8.1
64 bit packet counters: enabled
Verbose printing via tcpdump: enabled
Packet editing: disabled
Fragroute engine: enabled
Injection method: PF_PACKET send()
Not compiled with netmap
```


Exact command failed : 
`sudo tcpreplay --loopdelay-ms=600000 --stats=60 -q --loop=0 -i eth1 shoutcast.pcapng`
Workaround (increasing the loop):
`sudo tcpreplay --loopdelay-ms=600000 --stats=60 -q --loop=100000 -i eth1 shoutcast.pcapng`

**Comment by fklassen** [comment]
Fixed in PR #608
